src/UserSettingsStore.js

```diff
@@ -27,20 +27,17 @@ export default {
     LABS_FEATURES: [
         {
             name: "-",
-            id: 'rich_text_editor',
-            default: false,
-        },
-        {
-            name: "-",
+            _tName: "Matrix Apps", // Translated!
             id: 'matrix_apps',
             default: false,
         },
     ],
 
     // horrible but it works. The locality makes this somewhat more palatable.
     doTranslations: function() {
-        this.LABS_FEATURES[0].name = _t("New Composer & Autocomplete");
-        this.LABS_FEATURES[1].name = _t("Matrix Apps");
+        this.LABS_FEATURES.forEach((f) => {
+            f.name = _t(f._tName);
+        });
```

Comment:
Lumping this in with this change means it'll get reverted along with the rest of this PR if we end up finding showstopper bugs in RTE. Also it won't be picked up by my translations finder script which will only know about `_t(<string literal>)`